### DISCUSSION TITLE
r.report is function response onCopilot prompt response

### DISCUSSION THREAD

**Discussion by alikulaOwen** [community/community#125568] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

![Screenshot from 2024-05-24 14-51-49](https://github.com/community/community/assets/49692066/3e0c6a28-b388-473e-a59c-d2a55ac99c99)
I have been getting this response on copilot every time i add a propmpt that r.report is not a function. How do i fix it ? Can use it and i paid which is not due

**Comment by davevad93** [comment, marked as answer]
Hi @alikulaOwen ,

Try checking [this](https://github.com/microsoft/vscode-copilot-release/issues/1129) and [this](https://github.com/microsoft/vscode-copilot-release/issues/1194), apparently this is an issue that many users have fixed by either updating Copilot or reinstalling the extension in VS Code.

**Reply by alikulaOwen** [reply]
Let me give it a try. thank you

**Reply by davevad93** [reply]
You're welcome @alikulaOwen , I hope those tips can help you fix it.

**Reply by alikulaOwen** [reply]
Yeah fixed
